src/marqo/client.py

```diff
@@ -189,4 +196,13 @@ def get_cuda_info(self):
     def get_cpu_info(self):
         return self.http.get(path="device/cpu")
 
+    def _marqo_minimum_supported_version_check(self):
+        if self.url not in marqo_url_and_version_cache:
+            marqo_url_and_version_cache[self.url] = self.get_marqo()["version"]
+        marqo_version = marqo_url_and_version_cache[self.url]
+        if marqo_version < __minimum_supported_marqo_version__:
+            mq_logger.warning(f"Your Marqo python client requires a minimum Marqo version of "
+                              f"{__minimum_supported_marqo_version__} to function properly, while your Marqo version is {marqo_version}. "
+                              f"Please upgrade your Marqo instance to avoid potential errors. "
+                              f"If you have already upgraded your Marqo instance but still having this warning, please import you Marqo python client again and retry.")
```

Comment:
Python must be capitalised 

Your Marqo Python client requires a minimum Marqo version of